src/ExportMap.js

```diff
@@ -509,6 +511,32 @@ ExportMap.parse = function (path, content, context) {
         m.reexports.set(s.exported.name, { local, getImport: () => resolveImport(nsource) })
       })
     }
+
+    // This doesn't declare anything, but changes what's being exported.
+    if (n.type === 'TSExportAssignment') {
+      const moduleDecl = ast.body.find((bodyNode) =>
+        bodyNode.type === 'TSModuleDeclaration' && bodyNode.id.name === n.expression.name
+      )
+      if (moduleDecl && moduleDecl.body && moduleDecl.body.body) {
+        moduleDecl.body.body.forEach((moduleBlockNode) => {
+          // Export-assignment exports all members in the namespace, explicitly exported or not.
+          const exportedDecl = moduleBlockNode.type === 'ExportNamedDeclaration' ?
+            moduleBlockNode.declaration :
+            moduleBlockNode
+
+          if (exportedDecl.type === 'VariableDeclaration') {
+            exportedDecl.declarations.forEach((decl) =>
+              recursivePatternCapture(decl.id,(id) => m.namespace.set(
+                id.name, captureDoc(source, docStyleParsers, decl, exportedDecl, moduleBlockNode))
```

Comment:
if each item can’t fit on one line, please put each on its own line:
```suggestion
                id.name,
                captureDoc(source, docStyleParsers, decl, exportedDecl, moduleBlockNode))
```